github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1475,3 +1475,69 @@ periodics:
         resources:
           requests:
             memory: "29Gi"
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: prow-workloads
+  cron: 0 6 * * *
+  decorate: true
+  decoration_config:
+    grace_period: 5m0s
+    timeout: 1h0m0s
+  extra_refs:
+  - base_ref: main
+    org: kubevirt
+    repo: kubevirt
+  labels:
+    ci.kubevirt.io/prometheus: ""
+    preset-bazel-cache: "true"
+    preset-bazel-unnested: "true"
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+  name: periodic-kubevirt-e2e-k8s-1.22-sig-performance-realtime
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  spec:
+    containers:
+    - command:
+      - /usr/local/bin/runner.sh
+      - /bin/sh
+      - -c
+      - |
+        # Create k8s cluster
+        make cluster-up
+
+        # Push and deploy kubevirt
+        make cluster-sync
+
+        FUNC_TEST_ARGS="--noColor --seed=42" make realtime-perftest
+      env:
+      - name: KUBEVIRT_PROVIDER
+        value: k8s-1.22
+      - name: KUBEVIRT_STORAGE
+        value: hpp
```

Comment:
I'm not clear about how this is going to work. IIRC we only have support for values rook-ceph and rook-ceph-default in kubevirt/kubevirt and I didn't see that you did add it in the PR https://github.com/kubevirt/kubevirt/pull/6182? Or did I overlook a PR in kubevirt/kubevirtci? Could you point me to the right direction here please?

Reply:
Oh my bad then. I copied it from the [performance job](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L1225-L1226) thinking that it would be the closest example I could use. Should I change it then to `rook-ceph`?

Reply:
No, please use `rook-ceph-default` as that will use the embedded support from kubevirtci. `rook-ceph` would just work when using `automation/test.sh`.

Reply:
@awels or do we implicitly have support for `hpp` value for env `KUBEVIRT_STORAGE` that I am not aware of?

Reply:
No just rook-ceph-default.

Reply:
Done :smile: